### PR TITLE
linting changes

### PR DESCRIPTION
**Pull request by unclejack**
This PR makes some linting changes.

### CODE REVIEW

File: pkg/httputils/httputils.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/docker/docker/pkg/jsonmessage"
 )
 
-// Request a given URL and return an io.Reader
+// Download requests a given URL and return an io.Reader
```
Comment (thaJeztah) [inline pkg/httputils/httputils.go]: s/return/returns/

Comment (unclejack) [inline pkg/httputils/httputils.go]: Thanks, I've fixed it.

**Comment by LK4D4** [comment]
@unclejack There is import cycle on windows :/

**Comment by unclejack** [comment]
@LK4D4 What's up with the import cycle on Windows? I don't have a Docker setup on Windows to test.

**Comment by LK4D4** [comment]
@unclejack You need to rebase against master. It's fixed there.

**Comment by unclejack** [comment]
Ok, it seems to be OK. @LK4D4 Please let me know if there's anything else you'd like me to do.

**Comment by icecrime** [comment]
LGTM! Can you please squash commits?

**Comment by unclejack** [comment]
@icecrime Done, they were split by package.

**Comment by calavera** [comment]
LGTM, will merge when it turns :green_heart:

**Pull request merged**